src/Compatibility/ControlGallery/src/Core/Startup.cs

```diff
@@ -34,10 +34,7 @@ public static MauiAppBuilder CreateMauiAppBuilder()
 				{
 					handlers.AddCompatibilityRenderers(AppDomain.CurrentDomain.GetAssemblies());
 				})
-				.ConfigureImageSources(sources =>
-				{
-					sources.AddCompatibilityServices(AppDomain.CurrentDomain.GetAssemblies());
-				})
+				.AddCompatibilityServices(AppDomain.CurrentDomain.GetAssemblies())
```

Comment:
This used to be lazy, where it wouldn't call `AppDomain.CurrentDomain.GetAssemblies()` until you called `Build()` on the "MauiAppBuilder".

Does this fix something, or should we change it back?

Reply:
I removed `ConfigureImageSources(this, configureDelegate)` all together so ImageSourceServiceProvider could be removed completely, so I wanted to keep the AddCompatibilityServices logic.

Reply:
Is there another way to keep the logic as lazy if we are removing ImageSourceServiceProvider? the lazy logic worked previously because the ImageSourceServiceProvider would only be initialized once there was a call to `GetRequiredService<IImageSourceServiceProvider>()` right? This PR currently has the keyed services being registered directly to the MauiApp ServiceProvider, so no intermediate ImageSourceServiceProvider.

Reply:
For now, I think leave it as you have it here. We don't think there are many that would use this API.